drafts/use-cases/5.creating-new-event.md

```diff
@@ -0,0 +1,88 @@
+## Creating a new event
+
+### Story
+As an application user
+I want to be able to create new events manually
+So I can have event's I'm aware of added to my calendar.
+
+### Usage
+```javascript
+var event = {
+    "eventName": "My brand new event",
+    "eventDescription": "Hope it will work",
+    "startDate": "2017-04-19",
+    "endDate": "2017-04-19"
+};
+var client = new HydraClient();
+var operation = client.get("http://example.com")
+    .getApiDocumentation()
+    .getEntryPoint()
+    .operations
```

Comment:
This touches a topic of the internal client's data model. During my exercises with implementing a hydra client I had to use completely JSON-LD detached model.
The examples I'm using are not directly bound to the JSON-LD payloads as well. I can easily imagine _operations_ to be a property *getter*. It can also be a framed JSON-LD.